### PR TITLE
[T4A4][T16-B4] Yip Bing You

### PR DESCRIPTION
**Pull request by leaffriend**

### CODE REVIEW

**Comment by leaffriend** [comment]
Ready for Review

File: src/seedu/addressbook/data/person/Printable.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package seedu.addressbook.data.person;
+
+public interface Printable{
```
Comment (lohvht-zz) [inline src/seedu/addressbook/data/person/Printable.java:3]: missing header comments here

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -165,5 +166,19 @@ private static String getIndexedListForViewing(List<String> listItems) {
...
-
+    
+    private static String getPersonPrintableString(ReadOnlyPerson person) {
+    	return getPrintableString(person.getName(), person.getPhone(), person.getEmail(), person.getAddress());
+	}
+
+    private static String getPrintableString(Printable... printables){
+    	String line = "";
+    	for(Printable printItem: printables){
+    		line += printItem.getPrintableString();
```
Comment (lohvht-zz) [inline src/seedu/addressbook/ui/TextUi.java:177]: concatenating a string within a loop in this manner may not be the most efficient way of joining Strings. You may want to look into StringBuilder or other APIs to do this inside a loop

**Comment by lohvht-zz** [comment]
@leaffriend Added some comments, please close this PR after you're done with reading these comments

**Pull request closed without merging**